lib/host.rb

```diff
@@ -854,20 +854,36 @@ def exec_raw(*commands, **opts)
         commands = ["chroot", "/host/", "bash", "-c", commands_to_string(commands)]
       end
 
+      unless node.env.opts[:host_debug_image] == "support-tools"
```

Comment:
`if a !+ b` will read better here. `unless` is harder to understand.

In either case I would assume you want to set this multus image only in case `:host_debug_image` is not set to anything.

So perhaps here we should have:
```
unless node.env.opts[:host_debug_image]
```

Reply:
Or to avoid extra calls when we already have it
```
unless self[:debug_image] || node.env.opts[:host_debug_image]
```

Reply:
Wait, if we have `node.env.opts[:host_debug_image]`, then `self[:debug_image]` should be set to that value.

Reply:
Unless I misunderstood your intention.

Reply:
If we are using the multus image by default I wasn't sure if we wanted to allow the user to override the image to anything so I didn't want to use environment variable to set any random debug image, only to default back to the default `oc debug` image, whatever version of `support-tools` is hard-coded in the `oc` binary.

the plan was:

| HOST_DEBUG_IMAGE | image |
| --- | --- |
| nil | use multus |
| non-nil | use default `oc debug` image, current behavior  |

One problem with using the `multus` image is that is may disappear after an upgrade when a new multus image is loaded.  We could cache the multus value and try to invalidate the cache after an upgrade, but that seems to complex, so I decided to just re-query the current multus image each time we run.